src/groups/NewGroupModal.test.tsx

```diff
@@ -117,12 +120,12 @@ describe('NewGroupModal', () => {
       <NewGroupModal onDismiss={jest.fn()} onSuccess={mockOnSuccessFn} existingGroups={[]} />
     );
     const nameInput = getByLabelText('Enter a unique name *');
-    fireEvent.change(nameInput, { target: { value: 'ValidName' } });
-    await waitFor(() => expect(nameInput).toHaveValue('ValidName'));
+    await user.type(nameInput, 'ValidName');
```

Comment:
This is the case that I was warning about-- `user.type` is much slower than doing a fireEvent. You should go back to `fireEvent` for typing.

Reply:
https://github.com/DataBiosphere/terra-ui/pull/5066#discussion_r1752157814

Reply:
Wow, you're right: it is between 1-2 seconds, with just those usages, running locally.